### PR TITLE
Add a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by beaushinkle**
refs https://github.com/keep-network/pm/issues/3

This PR adds a CONTRIBUTING file, that way github will automatically link it for folks thinking about opening a PR.

In order to accomplish this, we add explicit instruction information, including node and python versions to get the project to compile and run properly.

tagging @pdyraga for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,34 @@ on the provided pseudo-random seed.
 Each privileged application has its own sortition pool and is responsible for
 maintaining operator weights up-to-date.
 
+## Setup
+
+### Prerequisites
+
+* clone the repository
+* link:https://nodejs.org/en/[Node.js] v14.21.2
```
Comment (dimpar) [inline README.md:18]: Do we need to replace [Node.js] with a version? This link redirects to 404

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+To run the linters and formatters manually, use the following commands:
+* `npm run lint` - checks files against linting rules.
+* `npm run lint:fix` - checks files against linting rules and fixes any
+  issues that can be fixed automatically.
+
+If you want to change a rule, or add a custom rule, to the JavaScript or
+Solidity linting, please propose these changes to our
+https://github.com/keep-network/solium-config-keep[solium-config-keep] and
+https://github.com/keep-network/eslint-config-keep[eslint-config-keep] packages.
```
Comment (dimpar) [inline CONTRIBUTING.adoc:54]: There's also [prettier-config-keep](https://github.com/keep-network/prettier-config-keep)

File: README.md [hunk 3]
```diff
@@ -10,6 +10,34 @@ on the provided pseudo-random seed.
 Each privileged application has its own sortition pool and is responsible for
 maintaining operator weights up-to-date.
 
+## Setup
```
Comment (dimpar) [inline README.md:13]: I know it was not part of this change but in the paragraph above it says
> In the Keep network the stake consists of staked KEEP tokens.

Is this relevant anymore? Shouldn't it be Threshold and T now?

File: CONTRIBUTING.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+=== Continuous Integration
+
+Keep uses https://github.com/keep-network/sortition-pools/actions[Github
+Actions] for continuous integration. All jobs must be green to merge a PR.
+
+=== Linting
+
+Linters and formatters for Solidity, JavaScript, and Go code are set up and run
+automatically as part of pre-commit hooks. These are checked again in CI builds
```
Comment (dimpar) [inline CONTRIBUTING.adoc:43]: It might be useful adding a paragraph on pre-commit hooks just like you did in other repos:

```
=== Pre-commit

Pre-commit is a tool to install hooks that check code before commits are made.
It can be helpful to install this, to automatically run linter checks and avoid
pushing code that will not be accepted. Follow the
https://pre-commit.com/[installation instructions here], and then run
`pre-commit install` to install the hooks.
```

File: README.md (hunk 1 above)
Comment (beaushinkle) [reply README.md:18]: good catch! the other readmes are in adoc and this one is in md (which I should probably also fix)

Comment (beaushinkle) [reply README.md:18]: [ea4a41](https://github.com/keep-network/sortition-pools/pull/202/commits/ea4a415cf6812012154fd8806457a673198fd400)

File: CONTRIBUTING.adoc (hunk 2 above)
Comment (beaushinkle) [reply CONTRIBUTING.adoc:54]: [b416a0](https://github.com/keep-network/sortition-pools/pull/202/commits/b416a0252db88d6ad58e60c9a2f463d0cab91c01)

File: CONTRIBUTING.adoc (hunk 4 above)
Comment (beaushinkle) [reply CONTRIBUTING.adoc:43]: [6d724d](https://github.com/keep-network/sortition-pools/pull/202/commits/6d724d884ef87d6fb5dd96993b87b028deee8836)

**Review by dimpar — APPROVED** [review]
